sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java

```diff
@@ -148,8 +198,48 @@ public CustomCaseExportCSV parseCustomCaseExport(String fileName) {
       builder =
           CustomCaseExportCSV.builder()
               .disease(values[0])
-              .firstName(values[1])
-              .lastName(values[2])
+              .firstName(String.format(values[1], Locale.GERMAN))
+              .lastName(String.format(values[2], Locale.GERMAN))
+              .build();
+    } catch (NullPointerException e) {
+      log.error("Null pointer exception parseCustomCaseExport: ", e);
```

Comment:
you need to add the {} so the string will be formatted and print e value ( also use e.getMessage() )